nexus/users/models.py

```diff
@@ -30,21 +32,49 @@ def create_superuser(self, email: str, password: str, **extra_fields):
 
 
 class User(AbstractBaseUser, UUIDModel, PermissionsMixin):
-    first_name = models.CharField(_('First Name'), max_length=120, blank=True)
-    last_name = models.CharField(_('Last Name'), max_length=120, blank=True)
+
+    GENDER_CHOICES = Choices(
+        ('NOT_SELECTED', 'NA', _('Not Selected')),
+        ('MALE', 'M', _('Male')),
+        ('FEMALE', 'F', _('Female')),
+        ('OTHERS', 'O', _('Others')),
+    )
+
+    TSHIRT_SIZE_CHOICES = Choices(
+        ('NOT_SELECTED', 'NA', _('Not Selected')),
+        ('SMALL', 'S', _('Small')),
+        ('MEDIUM', 'M', _('Medium')),
+        ('LARGE', 'L', _('Large')),
+        ('EXTRA_LARGE', 'XL', _('Extra Large')),
+        ('DOUBLE_EXTRA_LARGE', 'XXL', _('Double Extra Large')),
+    )
+
+    first_name = models.CharField(_('First Name'), null=False, blank=True, max_length=120)
+    last_name = models.CharField(_('Last Name'), null=False, blank=True, max_length=120)
     # https://docs.djangoproject.com/en/1.11/ref/contrib/postgres/fields/#citext-fields
-    email = CIEmailField(_('email address'), unique=True, db_index=True)
-    is_staff = models.BooleanField(_('staff status'), default=False,
+    email = CIEmailField(_('Email Address'), null=False, blank=False, unique=True, db_index=True)
+    is_staff = models.BooleanField(_('Staff Status'), default=False, null=False, blank=True,
                                    help_text='Designates whether the user can log into this admin site.')
 
-    is_active = models.BooleanField('active', default=True,
+    is_active = models.BooleanField('Active', default=True, null=False, blank=True,
```

Comment:
The same logical explanation as for the `is_staff` of why it's not necessary to include `null` and `blank`.